proposals/839-outreach-committee/kubeflow-outreach-committee.md

```diff
@@ -0,0 +1,31 @@
+# KEP- 839: Proposal for the Creation of the Kubeflow Outreach Committee (KOC)
```

Comment:
@franciscojavierarceo Please can you rename this file to README.md.
Also, could you move your other proposal to the dedicated directory (748-expand-kubeflow-ecosystem) please: https://github.com/kubeflow/community/blob/52334965db95f4f36942aa4fe34768ad7319dd05/proposals/new-project-join-process.md ?

You can find example how the KEPs should be structured here: https://github.com/kubernetes-sigs/kueue/tree/main/keps

Reply:
Yeah